test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-StringData.Tests.ps1

```diff
@@ -65,3 +65,50 @@ bazz = 2
 	$(ConvertFrom-StringData -StringData $sampleData).Values | Should -BeIn @("0","1","2")
     }
 }
+
+Describe "Delimiter parameter tests" -Tags "CI" {
+
+    $sampleData = @"
+
+a:b
+
+"@
+
+
+    It "Should return the data on the left side in the key" {
+        $actualValue = ConvertFrom-StringData -StringData 'a:b' -Delimiter ':'
+
+        $actualValue.Keys | Should -BeExactly "a"
```

Comment:
Test below with TestCases does the check so we can remove the test.

Reply:
Done!